### DISCUSSION TITLE
plotPODF_transform error

### DISCUSSION THREAD

**Discussion by NovakEvana** [ORTools4MTEX/ORTools#22] [Q&A] [answered]
Hello Frank,

Sorry for disturbing you again. I'm trying to produce a transformed texture from certain variants using `plotPODF_transform` but ended up with the below error. May you take a look at this please? I've posted my code (mostly based on example 4) and my dataset [here](https://drive.google.com/drive/folders/17pPBfv5NrmZhDJtZg16rgvlOFK1cowmW?usp=sharing). Thanks Frank and have a good weekend.

![image](https://user-images.githubusercontent.com/117882416/204115971-6bde6247-f9bf-403c-84d5-174260eb8ae8.png)

**Comment by frankNiessen** [comment, marked as answer]
Hi Evana, 

I see. I just ran your code and fund the bug, this time it is on MTEX's side. 
If you quickly want to fix it replace line 173 in variants.m.

Change:

`out = reshape(oriParent.project2FundamentalRegion,[],1) .* inv(p2cVariants);`

to 

`out = reshape(oriParent.project2FundamentalRegion,[],1) * inv(p2cVariants);`

I also updated the developer version of MTEX to correct for this. 
Thank you for finding the problem, let's hope there won't be more :).

Best wishes
Frank

**Reply by NovakEvana** [reply]
Really!? It's the most complicated code change I've ever seen in my life! It works perfectly. Thanks Frank!
